### PR TITLE
Register generic.is-a.dev

### PR DESCRIPTION
**Pull request by xgeneric**
Register generic.is-a.dev with CNAME record pointing to pandi.paulswebhelp.com.
(posted from Discord)

### CODE REVIEW

File: domains/generic.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "email": "",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.jL6LDyd2SQOHn6_qtQbQolYYL-9xQ0-jjEzNzMiteQUsdaa0HX8dVvkM21JcCuxEZbJPokrRdFQRAeeBrF45XMMg4NynXkwzJewMbGkY-oahqIcdm4LrX0h-gkbl-1zlnZJQP94JkjxBa93rnyMgGaX4DkCJpzpdrODlpxPDoTDPn7Mhg8rqU2nKhXoC1WD2fWTdQAAWrl5cBAT1VR0aAPISL5pzOrLMB3ldVxHCGcGBBEzsqW2EJRGY6UjAwQ0FE7IH-4wyewg4M6yQtjT-Cntz6Uxg5iBxA_bkCnsSjhQuE0PtfBkuIz-Pv2TF6QYHLUJdzJKs0lgYgeZGV4cHuQ.JplpmnheJVC0WyaVM3X4YA.udfWABodxYzHr_mFAj82UYRZqaySGLV-_EzZVuwkzG3X_J-GmEgGqkU6C6G6JabpJWpaO7pFzDTaPx49GirTzdGKCziFQ9pvFbOYYBx-QZxg8oZILvQY2886y8C2Pcr_.0Wh04zFffxpmI3dikZKF8Q"
+        },
+    
+        "record": {
+            "CNAME": "https://pandi.paulswebhelp.com/"
```
Comment (orxngc) [inline domains/generic.json:10]:
```suggestion
            "CNAME": "pandi.paulswebhelp.com"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).

File: domains/blooket.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "email": "",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.L3V3MOMY79wT_NK9PY8vOHLtKZRW2sxVsd-opDX1Wb3R3ZH72mB6C1FsVBSrbnGOJkC-OoEAq3xpLuSOm7KPzdIF9nExBRNltWyCVIGsZb_IGqrKLNxHNacD4Ls-XzVMZncux-ZXgd4-99kPg4NdXugZV34ugk38TQwuqwnRoLKMRjq4WjTP8sS3tidrSVjAZvcGbWbYQVeSo_IDjdGkFhw8Wsur4FFSGB7W2yBU6u1h4C6eYZYjHw5xPafXqzX-IDtMukfgnI1fXIjBgszRUOUuDQEDr4kwL-EecFeA9c1tQG3cQqGFbrt71fINIWq5CH2OsCBvi3dZboc2KsQ2vw.0MVCkLczAR3HwNs5rHRyag.F5TEtKQ3vAwxpynNWdsv0H4g2-Vs1zijaYtnFjM7muDSTQGa-bmwFkIejrlaOVvGuWhq1D4FQsQU_WnMwToCpEwfMPAbif8n80sQfqLj7LOMsD1uzFwE_iBfkdrj_uuI.EJMYfqdvwlqVh5q0GcGnaA"
+        },
+    
+        "record": {
+            "CNAME": "https://xgeneric.github.io/blooket/"
```
Comment (orxngc) [inline domains/blooket.json:10]:
```suggestion
            "CNAME": ":xgeneric.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).

**Review by orxngc — CHANGES_REQUESTED** [review]

File: domains/blooket.json (hunk 2 above)
Comment (CuteDog5695) [reply domains/blooket.json:10]: You had a typo, fixed that now

Comment (xgeneric) [reply domains/blooket.json:10]: > CNAME records cannot contain protocols (`https://`) or slashes (`/`).

thanks, fixed.

**Pull request closed without merging**